doc/start/os-recommendations.rst

```diff
@@ -43,7 +43,7 @@ package (i.e., sysvinit, systemd).
 +--------------+--------+------------------------+--------------------------------+-------------------+-----------------+
 | Release Name | Tag    | CentOS                 | Ubuntu                         | OpenSUSE :sup:`C` | Debian :sup:`C` |
 +==============+========+========================+================================+===================+=================+
-| Quincy       | 17.2.z | 8 :sup:`A`             | 20.04 :sup:`A`                 | 15.3              | 11              |
+| Quincy       | 17.2.z | 9 :sup:`1`             | 20.04 :sup:`A`                 | 15.3              | 11              |
```

Comment:
This doesn't make it clear what status/letter is applied to 9 IMO.

In main, `1` is a footnote on `A` -- it's not standalone.  Here, `1` appears to be just standing on itself.

Reply:
I have made it so that the text has a `9` followed by a superscript `A` followed by a superscript `1`.

Reply:
LGTM, although I don't know how it renders because Read the Docs check remains broken in quincy.